wiki/Tournaments/SOLMT/2019/en.md

```diff
@@ -0,0 +1,341 @@
+---
+tags:
+- SOLMT
+- SOLMT
+---
+
+# Speed of Light Mania Tournament
+
+![SOLMT logo](img/logo.png)
+
+The **Speed of Light Mania Tournament** (***SOLMT***) is a head to head (1vs1) based Mania tournament. It is the 1st installment of the Speed of Light Mania Tournament.
+
+## Tournament Schedule
+
+| Event | Timestamp |
+| --: | :-- |
+| Registration Phase |2019-10-14 / 2019-11-6|
```

Comment:
```suggestion
| Registration Phase | 2019-10-14/2019-11-6 |
```
Spacing between the dates, but don't give whitespace in the middle (to separate `/`)
Apply to all dates below